core/input/input.cpp

```diff
@@ -1164,6 +1166,29 @@ void Input::_button_event(int p_device, JoyButton p_index, bool p_pressed) {
 }
 
 void Input::_axis_event(int p_device, JoyAxis p_axis, float p_value) {
+	int axis_id = (int)p_axis;
+
+	if ((int)axis_to_button_mapper.size() > p_device && axis_id >= 0 && axis_id < (int)axis_to_button_mapper[p_device].size() && axis_to_button_mapper[p_device][axis_id].replace_axis_as_button) {
```

Comment:
Can `axis_id` ever be negative?

Reply:
yes:
https://github.com/godotengine/godot/blob/6916349697a4339216469e9bf5899b983d78db07/core/input/input_enums.h#L50-L51